Vagrantfile

```diff
@@ -1,11 +1,8 @@
-FLASK_PORT=5000
-POSTGRESQL_PORT=5432
-
 Vagrant.configure("2") do |config|
+
   config.vm.box = "ubuntu/bionic64"
   config.vm.provision :shell, path: "setup.sh"
-  config.vm.network :forwarded_port, guest: FLASK_PORT, host: FLASK_PORT
- config.vm.network :forwarded_port, guest: POSTGRESQL_PORT, host: POSTGRESQL_PORT
+  config.vm.network "private_network", type: "dhcp"
```

Comment:
you forgot to mention this change in your PR description

Reply:
Done

Reply:
well, they way you describe it there, tells people **what** you did - which should generally be obvious from the code, but not **why** you did it which is the one thing that code can never tell you on its own...

Reply:
Oh okay I didn't think about it, thank you. I'll rewrite it.

Reply:
@ifireball I referenced our discussion in the old PR.

Reply:
We'll personally I prefer to write a short summary of what I want people to know, instead of sending then to look for it in a discussion thread, but that is not a reason to block a PR.